scripts/one-offs/DR-3114-add-root-collection-uuid-string/add_root_collection_uuid_string.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: false
+
+# This comes from https://jira.nypl.org/browse/DR-3114
+
+# We need to update all records in the repoapi index to include the rootCollectionUUID_string field value, copied from rootCollectionUUID_s.
+
+# How to run (adjust solr url to the solr of your choice): ruby ./scripts/one-offs/DR-3114-add-root-collection-uuid-string/add_root_collection_uuid_string.rb http://10.225.133.217:8983/solr/repoapi
+
+require File.expand_path(File.join(File.dirname(__FILE__), '..', '..', '..', 'config', 'environment'))
+require 'rsolr'
+
+solr = RSolr.connect url: ARGV[0]
+
+solr_params = {
+    q: 'mainTitle:[* TO *] AND -rootCollectionUUID_string:[* TO *]',
+    rows: 500
+}
+
+response = solr.get 'select', params: solr_params
+
+total_found = response['response']['numFound']
+total_count = 0
+
+while total_count <= total_found do
+  update_json = []
+  response = solr.get 'select', params: solr_params
+  response['response']['docs'].each do |doc|
+    puts doc['uuid']
+    update_json << { uuid: doc['uuid'],
+    "dateIndexed_s" => { set: RepoSolrDoc.get_datetime_s },
+    "dateIndexed_dt" => { set: RepoSolrDoc.get_datetime_dt } }
```

Comment:
I'm probably misunderstanding the requirements (I don't know much about fedora 😅) , but looking at the comment on line 5, does a line that looks something like this need to be added? (apologies for any incorrect syntax)

`"rootCollectionUUID_string" => { set: doc['rootCollectionUUID_s'] }`

Reply:
It doesn't! Because it's a copyField, adding a line like that will throw an error.

Reply:
It gets populated automagically as the kids say.

Reply:
Oh gotcha, I think I see now! (if I'm looking in the right place):
<img width="1269" alt="Screenshot 2024-08-14 at 2 18 08 PM" src="https://github.com/user-attachments/assets/8dbe32d7-a036-4780-87c6-fd5b8521e619">

Very cool! 🎉

Reply:
Thank you! 🙌